source/percona-release.rst

```diff
@@ -42,20 +42,20 @@ official package:
 
    .. code-block:: bash
 
-      $ sudo apt-get update
-      $ sudo apt-get install -y wget gnupg2 lsb-release
+      $ apt-get update
+      $ apt-get install -y wget gnupg2 lsb-release
```

Comment:
```suggestion
      $ apt-get install -y wget gnupg2 lsb-release curl
```
As I remember @nastena1606 already added it previously but better to recheck

Reply:
Fixed.